### ISSUE TITLE
Data Review: Renew Places Maintenance Metrics

### ISSUE THREAD

**Issue by bendk** [mozilla/application-services#5387]
# 1. Provide a link to the initial Data Collection Review Request for this collection.

https://github.com/mozilla/application-services/issues/5122

# 2.  When will this collection now expire?

10-01-2023

This is a 6 month extension

# 3.  was the initial period of collection insufficient?

The maintenance feature has only been enabled for nightly/beta so far.  The Android team expects to enable it in production March 13, which is very close to the initial expiration date of 04-01-2023.

**Comment by eliserichards** [comment]
# Data Collection Renewal Review
1) Is the provided Data Collection Review complete, correct, and data-review+ by a Data Steward?
   * Yes.

2) Is the data collection covered by the existing Firefox Privacy Notice?
   * Yes.

**Comment by bendk** [comment]
I just realized that these metrics were added in 2 phases.  Here's the data review for the second phase:
https://github.com/mozilla/application-services/issues/5247

@eliserichards can this also be covered by the data review?

**Comment by eliserichards** [comment]
Yup! you did the correct thing, just add it to the data review request :)
